### ISSUE TITLE
Do a pass on info messages in the proxy

### ISSUE THREAD

**Issue by dlespiau** [clearcontainers/proxy#37]
We have a few messages here and there, but it's probably a good idea to add a few more to fully capture the life cycle of a vm, session and client  objects.

**Comment by jodh-intel** [comment]
+1. I even wonder if we want some "well-known" [1] messages that can be used for performance anaysis.

---
[1] - fixed-format messages that can be relied on - any changes to them would need to be discussed amongst the team to ensure "consumers" of such messages are updated in sync.

**Comment by dlespiau** [comment]
Yes, absolutely! That's one of the idea behind making the proxy the central place for logging messages, it'd then be possible to parse that output to capture timings or even have a bootchart-like svg with the processes involved.
